ui-tests/tests/timeline-slider.spec.ts

```diff
@@ -23,17 +23,51 @@ async function openNotebook(page: Page, notebookPath: string) {
   await page.waitForSelector('.jp-Notebook', { state: 'visible' });
 }
 
-test.describe('Open from Path', () => {
 
-  test('should fail if there are console errors', async ({ page, tmpPath }) => {
+const isTimelineEnv = process.env.TIMELINE_FEATURE || "0";
+const isTimeline = parseInt(isTimelineEnv)
+
+test.describe('Timeline Slider', () => {
+
+  isTimeline && test.use({ autoGoto: false })
+
+  test('should fail if there are console errors when opening from path', async ({ page, tmpPath }) => {
```

Comment:
I don't understand what this test does. It seems that you do nothing if `isTimeline`?

Reply:
There is no need to execute this test in the added timeline test environment (with the set baseurl)
but this is the test for this PR that was merged https://github.com/jupyterlab/jupyter-collaboration/pull/401